app/components/blacklight/start_over_button_component.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Blacklight
+  # This is a copy of the Blacklight::StartOverButtonComponent that adds group=true to the path
+  class StartOverButtonComponent < Blacklight::Component
+    def call
+      grouped_path = "#{start_over_path}?group=true"
```

Comment:
I think it would be better to add the `group=true` parameter in the `start_over_path` method where we're creating the path. Something like `h[:group] = true` above line 20.

Reply:
Done :)